src/v2/Components/NavBar/__tests__/NavBar.jest.tsx

```diff
@@ -62,17 +62,17 @@ describe("NavBar", () => {
   describe("desktop", () => {
     it("renders logged out items", () => {
       const wrapper = getWrapper()
-      expect(wrapper.html()).toContain("Log In")
-      expect(wrapper.html()).toContain("Sign Up")
+      expect(wrapper.html()).toContain("navbar.login")
```

Comment:
this is strange.. should it contain the key? feels like the test should be testing if the actual text is there.

something like
```
it("works in English", () => {
  expect(getWrapper().html()).toContain("Log In")
})
it("works in Greek", () => {
  someFuncToMockBrowserLang("gr")
  expect(getWrapper().html()).toContain("Σύνδεση")
})
```

Reply:
Yeah, this is something that we also discussed during the implementation... but if the goal here is that at some point the language experts should be capable of using a third-party service to update the translations as they wish, do we want the tests to fail when the content of text changes?

Reply:
i would say yes 🤔.

i guess we should support both? it would be great to be able to test the actual text we expand to, to test that languages are actually applied correctly, but also we should allow for key strings to be checked, though that can just be the `testID` 🤔. Im not set on one or the other.

Maybe we start with something and see how it goes. I just feel like checking just for the key might take us to a place where a translation is missing, and we never know, because our test is green, and our website says `navbar.signup` instead of `Sign Up`.

Reply:
Yeah, this is a super valid point
I added [1b51258](https://github.com/artsy/force/pull/9797/commits/1b512587d6c40577e8ad1fab8a578314a5b30237) to change the test to look for the strings instead of keys